src/Microsoft.OData.Core/JsonLight/ODataJsonLightResourceSerializer.cs

```diff
@@ -56,7 +57,7 @@ internal void WriteResourceSetStartMetadataProperties(ODataResourceSet resourceS
                 resourceSet,
                 isUndeclared);
 
-            if (typeName != null && !typeName.Contains(ODataConstants.ContextUriFragmentUntyped))
+            if (!IsNullOrContainsContextUriFragmentUntyped(typeName))

```

Comment:
I feel like this abstraction is overly specific. Is it commonly used?

Reply:
Used in 2 places - sync and async method - and it looked and felt brittle enough to the extent that I didn't feel comfortable repeating it. I however believe this is the kind of method that the compiler would inline.

Reply:
It felt like the name of the method does not provide a considerable level of abstraction beyond what the statement itself does. I mean it's a matter of preference/perception so I don't think you should change it, but if I would give an analogy.

I think this abstraction makes sense:
```c#
bool isAdultAge(int age) {
   return age >= 18;
}
```
but I'd feel like the following one is gratuitous and possibly unnecessary:
```c#
bool isGreaterThanEighteen(int x) {
   return x >= 18
}
```

Reply:
About the inlining, I think that's something that could be verified. But I also don't think it's a major issue at this point.

Reply:
@habbes I got rid of the superfluous methods